include/swift/ABI/MetadataValues.h

```diff
@@ -1554,6 +1554,12 @@ enum class RawLayoutFlags : uintptr_t {
 
   /// Whether or not this raw layout type was declared 'movesAsLike'.
   MovesAsLike = 0x2,
+  
+  /// Whether this raw layout type is bitwise borrowable.
+  ///
+  /// No raw layout types are yet, but should we change our mind about that in the future,
+  /// this flag is here.
+  BitwiseBorrowable = 0x4,
```

Comment:
Wouldn't all raw layout types not be bitwise borrowable? Can't we just unconditionally set this to false in the init function?

Reply:
I thought that having a flag to allow a raw layout type to be bitwise-borrowable in the future would be good future proofing, should we have a reason to do so later.